pkg/modulewriter/tfwriter.go

```diff
@@ -439,3 +447,43 @@ func orderKeys[T any](settings map[string]T) []string {
 	sort.Strings(keys)
 	return keys
 }
+
+func filterVarsByGraph(vars map[string]cty.Value, group config.DeploymentGroup, graph map[string][]config.ModConnection) map[string]cty.Value {
+	if graph == nil {
```

Comment:
IIUC this for testing only? It's kind of ugly, but we can keep it.

Reply:
Removed.